### PR TITLE
Fixed several item scripts

### PR DESCRIPTION
**Pull request by Atemo**
* **Addressed Issue(s)**: N/A

* **Server Mode**: renewal

* **Description of Pull Request**: 

`bSubSize` reduces all damage types, including magic.

### CODE REVIEW

File: db/re/item_db_etc.yml [hunk 1]
```diff
@@ -33833,17 +33826,6 @@ Body:
...
-    AegisName: Freeze_Dream
-    Name: Frozen Dream
-    Type: Etc
-    Trade:
-      NoDrop: true
-      NoTrade: true
-      NoCart: true
-      NoGuildStorage: true
-      NoMail: true
-      NoAuction: true
```
Comment (Lemongrass3110) [inline db/re/item_db_etc.yml:33846]: Is this intentional?

**Review by Lemongrass3110 — APPROVED** [review]
Rest looks good!

File: db/re/item_db_etc.yml (hunk 1 above)
Comment (Atemo) [reply db/re/item_db_etc.yml:33846]: I guess the branch need an update!

Comment (Everade) [reply db/re/item_db_etc.yml:33846]: Was part of my instance database update. It's a consumable.
https://github.com/rathena/rathena/commit/c7ec076a6c1073f4ab9435c5b572cb8368698d16

**Pull request merged**